ssr/progress-bar.js

```diff
@@ -42,12 +52,45 @@ export default class ProgressBar {
       const bytes = process.memoryUsage().heapUsed;
       out += ` | ${this.rJust(this.humanFileSize(bytes))}`;
     }
+
+    return out;
+  }
+
+  draw() {
+    if (this.quiet) {
+      return;
+    }
+    const out = this.getRow();
+
+    for (let i = 0; i < this.reservedRows - 1; i++) {
+      process.stdout.write("\n");
+      process.stdout.clearLine();
+    }
+    process.stdout.cursorTo(0, this.bottom);
     process.stdout.clearLine();
-    process.stdout.cursorTo(0);
     process.stdout.write(out);
+
+    process.stdout.cursorTo(0, this.bottom - this.reservedRows);
+    process.stdout.clearLine();
   }
 
   stop() {
+    if (this.quiet) {
+      return;
+    }
+    const out = this.getRow();
+
+    /**
+     * Idea here is to "return" back the reseved rows
```

Comment:
My intention is that since we've reserved bottom 5 rows during the progress bar's `update`, here we're cleaning it up by going back 5 rows and printing the final progress bar there